src/Microsoft.TestPlatform.Common/DataCollection/Interfaces/IDataCollectionManager.cs

```diff
@@ -42,7 +42,7 @@ internal interface IDataCollectionManager : IDisposable
         /// <summary>
         /// Raises SessionStart event to all data collectors configured for run.
         /// </summary>
-        /// <returns>Are test case level events required.</returns>
+        /// <returns>bolean value specifying whether test case events are subscribed by datacollectors. Based on this execution process will decide whether to send TestCaseStart and TestCaseEnd events to dataCollector process.</returns>
```

Comment:
> Out of proc
vstest.console waits until the `SessionStarted` call returns from the datacollector exe. In that call return, `false` indicates that no events are subscribed.